Vagrantfile

```diff
@@ -298,6 +310,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                   ENV["CONTIV_CLUSTER_STORE"] || "etcd://localhost:2379",
                   ENV["CONTIV_DOCKER_VERSION"] || docker_version,
                   ENV['CONTIV_NODE_OS'] || "",
+                  ENV['CONTIV_V2PLUGIN_NAME'] || "contiv/v2netplugin:0.1",
```

Comment:
I think we should probably use the netplugin version string as tag, instead of 0.1. But, this is ok for now.